src/apps/beefy/positions.test.ts

```diff
@@ -17,6 +112,78 @@ const vault: BaseBeefyVault = {
   subType: 'cowcentrated',
 } as BaseBeefyVault
 
+const expectedBeefyVault1 = {
+  address: '0x987654321',
+  availableShortcutIds: ['deposit', 'withdraw', 'swap-deposit'],
+  dataProps: {
+    cantSeparateCompoundedInterest: true,
+    claimType: 'rewards',
+    contractCreatedAt: '1970-01-01T03:25:45.000Z',
+    dailyYieldRatePercentage: 0.07400740957195229,
+    depositTokenId: 'arbitrum-one:0x123456789',
+    earningItems: [],
+    manageUrl: 'https://app.beefy.com/vault/vault1',
+    safety: undefined,
+    tvl: '98765',
+    withdrawTokenId: 'arbitrum-one:0x987654321',
+    yieldRates: [
+      {
+        label: 'yieldRates.earningsApy',
+        percentage: 31,
+        tokenId: 'arbitrum-one:0x123456789',
+      },
+    ],
+  },
+  displayProps: expect.any(Function),
```

Comment:
`displayProps`, `pricePerShare`, and `shortcutTriggerArgs` are returned as functions for Beefy earn positions. Wasn't sure how to check for a specific function implementation so just used `expect.any(Function)` instead.